### ISSUE TITLE
Problematic live region for main content

### ISSUE THREAD

**Issue by paulschreiber** [techmatters/terraso-web-client#514]
## Description
The main content for each page is wrapped in a live region (<div> with class="MuiContainer-root MuiContainer-maxWidthLg css-1sbt6pe" and aria-live="polite"). This implementation triggers incomprehensible and/or unexpected announcements when any page content is updated, for example:
• When a grid is sorted, the whole grid content is read out; and
• When a character is typed into an input field, it is spoken twice.

Remove the aria-live attribute from the main content container. Where a live region is required, it should be implemented as an empty container on page load that is populated with any status message or content update that needs to be conveyed to screen reader users.
